Source/santad/SNTPolicyProcessor.mm

```diff
@@ -347,15 +340,15 @@ - (nonnull SNTCachedDecision *)decisionForFileInfo:(nonnull SNTFileInfo *)fileIn
     certificateSHA256:nil
     teamID:teamID
     signingID:signingID
-    isProdSignedCallback:^BOOL {
+    platformBinaryState:(targetProc->is_platform_binary
+                           ? PlatformBinaryState::kRuntimeTrue
+                           : PlatformBinaryState::kRuntimeFalse)isProdSignedCallback:^BOOL {
```

Comment:
odd formatting, is that clang-format?

Reply:
Wow, that's pretty insane formatting... but yes it is clang-formatted. I removed the parens which seemed to be enough to fix it.